source/index.js

```diff
@@ -211,19 +215,19 @@ class CacheableLookup {
 	}
 
 	async query(hostname) {
+		this.stats.query++;
 		let source = 'cache';
 		let cached = await this._cache.get(hostname);
 
 		if (!cached) {
```

Comment:
```suggestion
		if (cached) {
			this.stats.cache++;
		} else {
```

Reply:
I don't think I understand this particular suggestion quite well on top of others. 
Do you want me to get rid of ```!``` and make a different ``` if``` statement?